src/NuGet.Core/NuGet.Packaging/PackagesConfigReader.cs

```diff
@@ -144,7 +144,7 @@ public IEnumerable<PackageReference> GetPackages(bool allowDuplicatePackageIds)
             {
                 string id = null;
                 if (!PackagesConfig.TryGetAttribute(package, "id", out id)
-                    || String.IsNullOrEmpty(id))
```

Comment:
The only changes in this file are non-functional, so you can probably just revert.